### DISCUSSION TITLE
Optionally provide a custom serializer/formatter when calling .SqlServerSave(...)

### DISCUSSION THREAD

**Discussion by mickvikt** [paillave/Etl.Net#469] [Ideas]
It would be great to be able to provide a custom serializer/formatter to be used when saving records to DB table with `SqlServerSave` extension method.

For example, we needed to avoid trimming of values when migrating an old legacy system, but I'm sure there could be a lot of other applicable cases.

**Comment by paillave** [comment]
Lot of things could be done for the SqlServer extensions. I planed some. But at some point, trying to make everything possible and impossible means rewritting some chunks of EF.
I'm not saying it will never be done but as this stage, it might be interesting to consider EF core+ETL.NET extensions for EF core to have such features for free.
